### PR TITLE
Update dependencies and instructions

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
selinux is a shim for the systen python-selinux package.
https://pypi.org/project/selinux/

The galaxy collection is pinned because there is a mix of info and facts use in agnosticd. facts was depricated and then removed in 3.0.0, so sticking to the last version that makes both available for now.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 env
 my_vars.yml
 secret.yml
+secret.ocp3.yml
+secret.ocp4.yml
```
Comment (jwmatthews) [inline .gitignore:5]: @jmontleon can you help me understand the need for secret.ocp3.yml and secret.ocp4.yml

Comment (jwmatthews) [reply .gitignore:5]: Please ignore me...sorry I thought this PR introduced them, I see you are just fixing it so they arent committed by accident, thank you!

https://github.com/konveyor/mig-agnosticd/pull/196 has the info I was looking for.

**Comment by jwmatthews** [comment]
Ran through OCP3 and OCP4 provisioning and worked for me.

Also checked deprovision worked for both OCP3 and OCP4

Thank you @jmontleon

**Review by jwmatthews — APPROVED** [review]

**Pull request merged**